plasmapy/physics/parameters.py

```diff
@@ -1154,28 +1154,29 @@ def Debye_number(T_e: u.K, n_e: u.m**-3):
     return N_D.to(u.dimensionless_unscaled)
 
 
+
 @utils.check_quantity(
     n={'units': u.m ** -3, 'can_be_negative': False}
     )
-def inertial_length(n: u.m**-3, particle='e-'):
-    r"""Calculate the particle inertial length. At this length, the Hall effect
-    becomes important.
+@atomic.particle_input(require='charged')
+def inertial_length(n: u.m**-3, particle: atomic.Particle):
+    r"""
+    Calculate a charged particle's inertial length.
 
     Parameters
     ----------
     n : ~astropy.units.Quantity
-        Particle number density in units convertible to m**-3.
+        Particle number density in units convertible to
+        m\ :superscript:`-3`\ .
```

Comment:
There's a trade off here between having our docs pretty and readable from sphinx (which this accomplishes beautifully) and from the embedded docs, Python help function, Ipython/Jupyter's beautiful `inertial_length?` magic... which this superscript change is going to make worse :( Do you think `m**-3` is not a reasonable compromise?